.env.example

```diff
@@ -0,0 +1,10 @@
+# set to 1 or '1" ( default ) : install the dependencies during installation
+# set to 0 or "0": doesn't install the dependencies during installation
+# Be careful, if set to 0, the dependencies won't be present in the package.json.
+INSTALL_DEPENDENCIES = '1'
```

Comment:
Let's probably use true/false (it doesn't have to be in quotes), cause in the end we want to use this as a boolean value.